src/model/OWL.pl

```diff
@@ -382,13 +406,13 @@
   is_all_disjoint_classes(DC),
   tripledb_ask(DC,owl:members,RDF_list),
   is_rdf_list(RDF_list,List),
-  once((
+  (
     member(Sup_A,List), subclass_of(A,Sup_A),
```

Comment:
I think this can be done more efficient.
asserting that ground(A) (which seems to be the case), it should be ok to use:

    once((member(Sup_A,List), subclass_of(A,Sup_A)))

as if there is such a Sup_A, then the rest of the classes in List must be disjoint and thus A cannot be a subclass.